src/TraceEvent/Stacks/Stacks.cs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.  All rights reserved
 // This file is best viewed using outline mode (Ctrl-M Ctrl-O)
 
+using Microsoft.Diagnostics.Tracing.AutomatedAnalysis;
```

Comment:
@marshall-lerner, can you confirm that this using is needed?  Looking at the code diff, I don't think it is required (but could be wrong).

Reply:
You're right! Removing it now :)